staging/src/k8s.io/apiserver/pkg/storage/interfaces.go

```diff
@@ -246,5 +246,5 @@ type Interface interface {
 // If Begin is empty, it is left unbounded.
 // If End is empty, it is right unbounded.
 type Interval struct {
-	Begin, End string
+	Begin, End, Whitelist string
```

Comment:
Whilelist does need to have a copy for each interval

Reply:
There is no code reads whilelist from config file or initailze it somewhere.

Reply:
For flexibility, its tenant can be default, system or something else

Here is the code to initialize whitelist
if size >= 4 {
			m[strings.TrimSpace(slices[0])] = storage.Interval{Begin: strings.TrimSpace(slices[1]), End: strings.TrimSpace(slices[2]), Whitelist: strings.TrimSpace(slices[3])}
		}